### PR TITLE
Improve README.md quality

### PR DESCRIPTION
**Pull request by sks444**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# improve-mess-food-in-india        <a href="https://gitter.im/good-food-project">Gitter</a>
+# good-food-project
+
+## *Improve Mess Food in India*
```
Comment (RaiVaibhav) [inline README.md]: No italics please :stuck_out_tongue:

File: README.md [hunk 2]
```diff
@@ -8,14 +12,63 @@
...
+Like any open source project the Goal of this project will remain a goal until you contribute to it :)
+
+### Prerequisites
+
+* Python
+* Django
+
+### Setup
+
+```
```
[the remark this answers is not in the document]
Comment (sks444) [reply README.md]: @RaiVaibhav; see this before reviewing https://github.com/sks444/good-food-project/tree/working#good-food-project

[the remark this answers is not in the document]
Comment (RaiVaibhav) [reply README.md]: what to see there :/

[the remark this answers is not in the document]
Comment (RaiVaibhav) [reply README.md]: your backticks don't like a `*` to act like a  point. (like others)

[the remark this answers is not in the document]
Comment (sks444) [reply README.md]: Oh man, i have done it on purpose, it looks good in a box.

[the remark this answers is not in the document]
Comment (RaiVaibhav) [reply README.md]: read my comment 
>why backticks here ?
I am asking here, not reviewing here , not using `ack` or `unack` , 
github gives us three options `Comment`, `Approve`, `Request Changes` and I am hoping you know what `comment` means.

[the remark this answers is not in the document]
Comment (sks444) [reply README.md]: Thanks, can you approve it? :)

**Pull request merged**